index.html

```diff
@@ -26,9 +26,25 @@
     </nav>
   </div>
 </header>
+<section>
+  <div class="max-width intro">
+    <h1 class="intro-title">Vision Capital</h1>
+    <h2 class="intro-subtitle">The right way to fund and incubate your DeFi startup</h2>
```

Comment:
I think it would be better to use `p` here, because on this page we have much larger titles, but smaller than `h1`, and they should be `h2`, I guess